bedrock/settings/base.py

```diff
@@ -569,15 +564,6 @@ def lazy_email_backend():
                          'tl_PH', 'tr_TR', 'uk_UA', 'vi_VN', 'zh_CN',
                          'zh_HK', 'zh_TW']
 
-
-# FACEBOOK_TAB_URL is lazily evaluated because it depends on the namespace
-# and app ID settings in local settings.
```

Comment:
@pmclanahan - is there anything else we need to tidy up here? (do different envs havespecific app id settings?)